client/x11/Makefile.am

```diff
@@ -40,12 +40,12 @@ ibus_x11_LDADD = \
 	$(libIMdkit) \
 	$(libibus) \
 	@X11_LIBS@ \
-	@GTK2_LIBS@ \
+	@GTK3_LIBS@ \
```

Comment:
I'm not sure if everyone can accept this change.
Would you able to add if ENABLE_GTK3 ?

if ENABLE_GTK3
ibus_x11_LDADD += @GTK3_LIBS@
endif

if ENABLE_GTK2
ibus_x11_LDADD += @GTK2_LIBS@
endif